test/integration/buyers/accounts_controller_test.rb

```diff
@@ -391,4 +391,18 @@ class NotLoggedInTest < ActionDispatch::IntegrationTest
       assert_redirected_to provider_login_path
     end
   end
+
+  class ManyAccountsTest < ActionDispatch::IntegrationTest
+    def setup
+      @provider = FactoryBot.create(:provider_account)
+      login! @provider
+    end
+
+    test 'no N+1 problems in the index' do
+      populate = ->(n) { FactoryBot.create_list(:buyer_account, n, provider_account: @provider) }
+      assert_perform_linear_number_of_queries(slope: 3, populate: populate, matching: /users/) do
```

Comment:
Shouldn't we `assert_perform_constant_number_of_queries`? Why aren't all buyers queried by a single query?

Just asking, I didn't look at the code. Just this is how I would expect endpoint to work.

Reply:
btw in the past I had to use `warmup` to avoid issues caused by caching or who knows what.

Reply:
I think you're right and I misunderstood this. But I don't understand why the amount of queries to the users table keeps incrementing linearly after solving the N+1 and bullet stopped complaining. It's possible that there are more N+1 problems in this screen not detected by bullet?. I'll investigate further.